src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -410,7 +410,8 @@ public boolean configure(StaplerRequest request, JSONObject object)
 
               containerInfo = new MesosSlaveInfo.ContainerInfo(
                   containerInfoJson.getString("type"),
-                  containerInfoJson.getString("dockerImage"), volumes);
+                  containerInfoJson.getString("dockerImage"), volumes,
+                  containerInfoJson.getBoolean("dockerPrivilegedMode"));
```

Comment:
Don't understand this style. Either width not enough, either volumes should be on new line...